src/qs8-vadd/rvv.c.in

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <assert.h>
+#include <riscv_vector.h>
+#include <xnnpack/vbinary.h>
+$assert DATATYPE in ["QS8", "QU8"]
+$XINT8_T = {"QS8": "int8_t", "QU8": "uint8_t"}[DATATYPE]
+$VSIZE1 = {1: "m1", 2: "m2"}[LMUL]
+$VSIZE2 = {1: "m2", 2: "m4"}[LMUL]
+$VSIZE4 = {1: "m4", 2: "m8"}[LMUL]
+
+void xnn_${DATATYPE.lower()}_vadd_minmax_ukernel__rvv_u${LMUL}v(
+    size_t batch,
+    const ${XINT8_T}* input_a,
+    const ${XINT8_T}* input_b,
+    ${XINT8_T}* output,
+    const union xnn_${DATATYPE.lower()}_add_minmax_params params[restrict XNN_MIN_ELEMENTS(1)])
+{
+  assert(batch != 0);
+  assert(batch % sizeof(${XINT8_T}) == 0);
+  assert(input_a != NULL);
+  assert(input_b != NULL);
+  assert(output != NULL);
+
+  const ${XINT8_T} a_zero_point = params->rvv.a_zero_point;
```

Comment:
local variables we usually start with the letter v for value/vector.  e.g. va_zero_point.
Its not a vector (good) but we usually do this for scalar microkernel values as well.  optional